src/pages/Landing.jsx

```diff
@@ -1,19 +1,113 @@
 import { Todo } from "../components/Todo/Todo";
 import "./landingpage.css";
+import { useEffect, useReducer } from "react";
+import { reducerFunc } from "../reducers/landingPageReducer";
 
 function Landing() {
+	const initialState = {
```

Comment:
make this state global otherwise it will reload everytime when we goes on landing page